app/controllers/v2/repositories_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module V2
+  # Handles all requests for repository CRUD operations
+  class RepositoriesController < ApplicationController
+    def index
+      @collection = Repository.all
+      render json: @collection, each_serializer: V2::RepositorySerializer
+    end
+
+    def show
+      retrieve_resource
+      render_resource
+    end
+
+    def create
```

Comment:
We should probably find a way to easily adhere to the return codes specified in the jsonapi.org spec (here: http://jsonapi.org/format/#crud-creating-responses). Do you know of an good way to share this status logic between all the controllers we will write?

This of course applies to all other actions as well.